.github/workflows/azure-webapps-node.yml

```
@@ -22,8 +22,14 @@ PROJECT_NAME: 'content-creation'    # set this to your project name
 LOCATION_CODE: 'euw'               # set this to your location code
 LOCATION: 'westeurope'             # set this to your preferred location
 
+# Feature flags
+FEATURE_BUILD: true
```

actionlint (1.7.4)

unexpected key "FEATURE_BUILD" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)